common.go

```diff
@@ -52,26 +51,24 @@ func GatherMetadata(wh *workloads.WorkloadHelper, alerting bool) error {
 		return err
 	}
 	// When either indexing or alerting are enabled
-	if alerting && wh.MetricsEndpoint == "" {
-		wh.PrometheusURL, wh.PrometheusToken, err = wh.MetadataAgent.GetPrometheus()
+	if alerting && wh.Config.MetricsEndpoint == "" {
+		wh.Config.PrometheusURL, wh.Config.PrometheusToken, err = wh.MetadataAgent.GetPrometheus()
 		if err != nil {
 			return fmt.Errorf("error obtaining Prometheus information: %v", err)
 		}
 	}
-	wh.Metadata.ClusterMetadata, err = wh.MetadataAgent.GetClusterMetadata()
+	wh.ClusterMetadata, err = wh.MetadataAgent.GetClusterMetadata()
 	if err != nil {
 		return err
 	}
-	wh.Metadata.UUID = wh.UUID
-	wh.Metadata.Timestamp = time.Now().UTC()
-	wh.Metadata.MetricName = clusterMetadataMetric
```

Comment:
I think we can remove the clusterMetadataMetric constant

Reply:
Done.